File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        a = []
        for i in range(len(nums)):
            sum =1
            for j in range(len(nums)):
                if i != j:
                    sum *= nums[j]
            a.append(sum)
        return a

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        if(nums.length == 0){
            return 0;
        }
        HashSet<Integer> nset = new HashSet<>();
        for(int i=0;i<nums.length;i++){
            nset.add(nums[i]);
        }
        int lcs = 1;
        for(int n:nset){
            if(nset.contains(n-1)){
                continue;
            }
            else{
                int currentNumber = n;
                int currentLCS = 1;
                while(nset.contains(currentNumber+1)){
                    currentNumber++;
                    currentLCS++;
                }
                lcs = Math.max(lcs,currentLCS);
            }
        }
        return lcs;
    }
}
